common/src/main/java/com/wynntils/overlays/RaidProgressOverlay.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.overlays;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.overlays.OverlayPosition;
+import com.wynntils.core.consumers.overlays.TextOverlay;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.VerticalAlignment;
+
+public class RaidProgressOverlay extends TextOverlay {
+    private static final String TEMPLATE =
+            "{concat(\"§6§l§n\";current_raid;\"\\n\\n§dChallenge 1: \";if_str(eq(raid_room_time(\"challenge_1\");-1);\"§7--:--\";concat(\"§b\";leading_zeros(int(div(raid_room_time(\"challenge_1\");60));2);\":\";leading_zeros(int(mod(raid_room_time(\"challenge_1\");60));2)));\"\\n§dChallenge 2: \";if_str(eq(raid_room_time(\"challenge_2\");-1);\"§7--:--\";concat(\"§b\";leading_zeros(int(div(raid_room_time(\"challenge_2\");60));2);\":\";leading_zeros(int(mod(raid_room_time(\"challenge_2\");60));2)));\"\\n§dChallenge 3: \";if_str(eq(raid_room_time(\"challenge_3\");-1);\"§7--:--\";concat(\"§b\";leading_zeros(int(div(raid_room_time(\"challenge_3\");60));2);\":\";leading_zeros(int(mod(raid_room_time(\"challenge_3\");60));2)));\"\\n\\n§4Boss: \";if_str(eq(raid_room_time(\"boss_fight\");-1);\"§7--:--\";concat(\"§b\";leading_zeros(int(div(raid_room_time(\"boss_fight\");60));2);\":\";leading_zeros(int(mod(raid_room_time(\"boss_fight\");60));2)));\"\\n\\n§5Total: \";concat(\"§b\";leading_zeros(int(div(raid_time;60));2);\":\";leading_zeros(int(mod(raid_time;60));2)))}";
```

Comment:
We may not do this for other text overlays (my bad!), but if you have this template saved with actual new-lines, I think we should start using [Java Text Blocks](https://docs.oracle.com/en/java/javase/15/text-blocks/index.html) to include the actual, multi-line version if this, if we ever need to edit it, or if someone wants to use this as their base for their function.

Reply:
Edit: I see you already used text blocks below :)